standard/documentation-comments.md

```diff
@@ -61,6 +61,16 @@ Although developers are free to create their own set of tags, a recommended set
 
 Note carefully that the documentation file does not provide full information about the type and members (for example, it does not contain any type information). To get such information about a type or member, the documentation file must be used in conjunction with reflection on the type or member.
 
+A partial type or a partial non-type member (such as a method) can be declared in multiple parts, each of which can be in one or more compilation units, and each of which can have one or more documentation comments. A partial non-type member typically has a “defining partial declaration” and an “implementing partial declaration.”
+
+For a partial type, the document comments that apply directly to that type, if any, from each of its parts, are all written to the documentation file in some unspecified order.
+
+For a partial non-type member
```

Comment:
Colon at the end of this introduction?

(And again, maybe just partial method.)

Reply:
It's not an intro. It covers partial types only while the following para starts the list for partial methods.